runtime/compiler/control/JITServerHelpers.cpp

```diff
@@ -943,6 +943,11 @@ JITServerHelpers::postStreamFailure(OMRPortLibrary *portLibrary, TR::Compilation
       compInfo->getPersistentInfo()->setServerUID(0);
       _serverAvailable = false;
 
+      // Ensure that log files are not supressed if methods are now
+      // going to be compiled locally
+      if (TR::Options::requiresDebugObject())
+         TR::Options::suppressLogFileBecauseDebugObjectNotCreated(false);
+
```

Comment:
It's possible that the JITServer comes back online. We don't seem to take the reverse action.

Reply:
It gets reset when a remote compile request is made:
https://github.com/eclipse-openj9/openj9/blob/5a4ab53779ef5aa6e623d0ef4f34bcec4530683e/runtime/compiler/control/CompilationThread.cpp#L8621-L8626